front/src/common/BootstrapSNCF/SelectImprovedSNCF.tsx

```diff
@@ -105,9 +108,13 @@ function SelectImproved<T extends string | SelectOptionObject>({
 
   const optionsComponents = useMemo(
     () =>
-      filteredOptions.map((option) => (
-        <span className="select-menu-item" role="listitem" key={getOptionValue(option)}>
-          <button type="button" onClick={() => selectItem(option)}>
+      filteredOptions.map((option, index) => (
```

Comment:
Each option should be unique, you should remove the duplicates before giving the options to the component and revert these change